.changeset/thirty-carrots-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components': patch
+---
+
+Refactored property checking utility functions to allow more focused unit testing.
```

Comment:
This is not a relevant change for end users, so I'd not create a changeset (https://github.com/swisspost/design-system/blob/main/CONTRIBUTING.md#versioning)